streams/src/main/java/org/apache/kafka/streams/state/internals/SegmentedBytesStore.java

```diff
@@ -114,7 +125,18 @@
          * @param to            ending time range
          * @return
          */
-        HasNextCondition hasNextCondition(final Bytes binaryKey, long from, long to);
+        HasNextCondition hasNextCondition(final Bytes binaryKey, final long from, final long to);
```

Comment:
Could we remove this method and just have all the `fetch` calls use the other one?